### PR TITLE
Ft payl initial payload proc functions

### PR DESCRIPTION
**Pull request by tyrelkostyk**
Initial creation of payload processing functions. Contains but one function that rapidly polls and aquires data from each of the 8 sensors on both dosimeter boards. Relies on external functions from housekeeping team to send/recieve i2c commands/data, and to store payload data in memory.

### CODE REVIEW

File: payload-processing/payload_processing.h [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/*  payload_processing.h
+
+	Created by Tyrel Kostyk on February 11th 2020
+ */
+
+
+#define DOSIMETER_REFERENCE_VOLTAGE	(float) 3.30;
```
Comment (MackenzieStanger) [inline payload-processing/payload_processing.h]:
```suggestion
#define DOSIMETER_REFERENCE_VOLTAGE	( (float) 3.30 )
```
Macros don't need semi colons. I would also wrap it in brackets just to be safe

**Review by MackenzieStanger — CHANGES_REQUESTED** [review]

File: payload-processing/payload_processing.h (hunk 1 above)
Comment (MackenzieStanger) [reply payload-processing/payload_processing.h]: Also should probably add the documentation comments in if you have time. Other than that the code looks good to me

**Review by rarohel — COMMENTED** [review]
I'm confused by the conversion bytes, is the "high" byte supposed to be the top 8 bits and the "low" byte the bottom 8? It just seems a little confusing, perhaps some more documentation might be helpful.

**Review by rarohel — APPROVED** [review]
Awesome, looks good!

**Review by MackenzieStanger — APPROVED** [review]

**Pull request merged**